src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -12,6 +14,19 @@ trait SampleWdl {
   def wdlJson: WdlJson = {
     "{" + rawInputs.collect { case (k, v) => s""" "$k": "$v"""" }.mkString(",\n") + "}"
   }
+
+  def createCannedFile(prefix: String, contents: String, dir: Option[Path] = None): File = {
+    val suffix = ".out"
+    val file = dir match {
+      case Some(path) => Files.createTempFile(path, prefix, suffix)
```

Comment:
They're overloaded variants of the same function. It just makes me a sad panda that `path` is the only difference between those lines and there's not much we can do about it.